frontend/src/components/pages/CampOverview/EditCamperModal/EditCamperFormResponseSection.tsx

```diff
@@ -0,0 +1,170 @@
+import React from "react";
+
+import {
+  VStack,
+  RadioGroup,
+  Stack,
+  Radio,
+  Textarea,
+  CheckboxGroup,
+  Checkbox,
+  FormLabel,
+  FormControl,
+} from "@chakra-ui/react";
+import { FormQuestion } from "../../../../types/CampsTypes";
+
+const EditCamperFormResponseSection = ({
+  formQuestions,
+  camperResponses,
+  setFormResponses,
+}: {
+  formQuestions: FormQuestion[];
+  camperResponses: Map<string, string>;
+  setFormResponses: (formResponses: Map<string, string> | undefined) => void;
+}): JSX.Element => {
+  const updateFormResponse = (question: string, option: string) => {
+    const newCamperResponse = new Map(Object.entries(camperResponses));
+    newCamperResponse.set(question, option);
+    setFormResponses(newCamperResponse);
+  };
+
+  const renderMCQ = (
+    question: string,
+    answer: string | undefined,
+    options: string[],
+    questionId: string,
+    isRequired: boolean,
+  ) => {
+    return (
+      <FormControl key={questionId} isRequired={isRequired}>
+        <FormLabel>
+          <b>{question}</b>
```

Comment:
Whereever you use `<b>`, I would reccomend using Chakra's `<Text>` element instead, and passing in some props regarding weight just so we stick to Chakra whereever possible.

Reply:
done